pydantic/_internal/_config.py

```diff
@@ -193,6 +197,10 @@ def core_config(self, title: str | None) -> core_schema.CoreConfig:
                 code='validate-by-alias-and-name-false',
             )
 
+        # ser_json_timedelta will be deprecated in V3, so we patch the value to ser_json_datetime if set.
+        if (ser_json_timedelta := config.get('ser_json_timedelta')) is not None:
+            config['ser_json_datetime'] = ser_json_timedelta
```

Comment:
This seems wrong, `ser_json_datetime` has much bigger effect than `ser_json_timedelta`. I think if anything the reverse should be possible (`ser_json_datetime` will imply setting `ser_json_timedelta` if that's not set).